types/mermaid/mermaidAPI.d.ts

```diff
@@ -256,8 +262,17 @@ declare namespace mermaidAPI {
          */
         gantt?: GanttConfig | undefined;
 
-        class?: any;
-        git?: any;
+        journey?: any; // [todo]
```

Comment:
Are these // TODO comments necessary?

Reply:
Keeping these types as any isn't right, but it's a temporary solution. I currently have no time, and would submit a PR in future for this. Keep these todo comment is nothing wrong IMO.